src/sonic-yang-models/yang-models/sonic-interface.yang

```diff
@@ -68,6 +68,12 @@ module sonic-interface {
 					}
                                         default "0";
 				}
+				leaf mpls {
+					description "Enable/disable MPLS routing for the interface";
+					type string {
+						pattern "enable|disable";
```

Comment:
Default value for all the interface types?

Reply:
@kperumalbfn I looked at the types of tests currently under sonic-yang-models.  It looked like it was mostly parameter validation.  Since this mpls knob has two fixed values, I wasn't sure what more needed to be checked for that.  Do you have something specific in mind?  Can you provide a pointer to an example?

Reply:
@kperumalbfn I don't think we want to add a default value here, do we?  What happens if the platform does not support this attribute?  I think we do not want to explicitly set the attribute in that case.